Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json

```diff
@@ -2856,6 +2870,21 @@
                         }
                     }
                 }
+            },
+            "VersionPatch": {
+                "type": "object",
+                "required": ["versionID", "displayName", "patch"],
+                "properties": {
+                    "versionID": {
```

Comment:
Something that occurred to me- I think we'll need to specify a version identifier and display name for the base RenderNode as well.

Maybe we make a new `Version` object that includes `identifier` and `displayName` properties? And then include that in the RenderNode's `metadata` as well as in each `VersionPatch`. What do you think?

Reply:
I'm going back and making these changes and noticed you resolved this comment--just making sure we're still doing this?